.editorconfig

```diff
@@ -1,6 +1,6 @@
 root = true
 
-[*.{py,c,cpp,h,js,rst,md,yml}]
+[*.{py,c,cpp,h,js,rst,md,yml,yaml}]
```

Comment:
Maybe we should instead rename `yaml` files to `.yml`?

Reply:
Only `.yaml` is supported. https://github.com/pre-commit/pre-commit/blob/main/pre_commit/constants.py#L5

Reply:
Oh :(

Reply:
Nevermind, actually `.yaml` is preferred over `.yml` according to the RFC: https://www.rfc-editor.org/rfc/rfc9512.html#section-2.1-2.22.2.6.